### PR TITLE
#175494059 Login with Email and Password

### PR DESCRIPTION
**Pull request by LouangeMu**
### What does this PR do?
This PR is going  is going to help a user to login with Email and Password
#### Description of Task to be completed?
- When a user visits the application landing page and clicks on the "Login" button, they should be able to login using their email and password. They should subsequently be routed to the next page.

#### How should this be manually tested?

- open your terminal
- run git clone `https://github.com/Soma-Technologies-Inc/visitAfrica-backend.git to clone this project`
- run git checkout `ft-Login-with-email-and-password-175494059`
- run `git pull `to pull all codebase from my branch
- run `npm install `to install all packages
- setup a `.env` file using `.env.example` format
- run `npx sequelize db:migrate`
- run `npm run startDev` to start the server
- open your  postman then make a request to this route of creating user `POST /api/auth/signup` and create a new user
-  Go to your postman and login using this path `http://localhost:3000/api/auth/login`
- -[#175494059](https://www.pivotaltracker.com/story/show/175494059)

#### Any background context you want to provide?
 
N/A

#### Screenshots (if appropriate)
**When the user is already in the database**
U1

**When the user is not in the database**
U2

**When the user has entered a wrong email**

u3

**When the user has one unfilled space**

image

**When the user has not entered any credential**

image

#### Questions:
N/A

### CODE REVIEW

**Comment by tigthor** [comment]
- work on the pr Description of how this should be tested add adequate information
- You should not that the reason why your tests are failing is that you are requiring login data that are not in the database so before you require data in the database you need to have them there . You can use different methods of entering userdata in the database like using signup route on your test and create a new user or seeding the user in the database.

File: preview.html [hunk 1]
```diff
@@ -1,25 +1,25 @@
-
```
Comment (tigthor) [inline preview.html:1]: This file is not needed

**Review by tigthor — CHANGES_REQUESTED** [review]
After working on other comments I left on your pr remember also to squash your commits into 1

**Review by tigthor — APPROVED** [review]
LGTM

**Comment by Niyonsengaeric** [comment]
LGTM!

**Review by Niyonsengaeric — APPROVED** [review]

**Pull request merged**